src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,246 @@
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed } from 'discord.js';
+import { LongLivingReactionCollector, LLRCData } from '../../lib/util/LongLivingReactionCollector';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Emojis } from '../../lib/util/constants';
+import { getColor } from '../../lib/util/util';
+
+enum ReactionEmoji {
+	HIGHER = '⬆',
+	LOWER = '⬇',
+	CANCEL = '❌', // TODO: Replace with <:redCross:637706251257511973>
+	OK = '✔️', // TODO: Replace with <:greenTick:637706251253317669>
```

Comment:
```suggestion
	OK = Emojis.GreenTick,
```

Reply:
This does not work as TS doesn't allow computer enums with string literals.